File: array.js
// ARRAY CRUD
const students = ['David', 'Risako', 'Andrei'];

// Create
students.push("Kim");
students.push("Amane");
console.log(students);
// Read
console.log(students[3]);
// Update
students[4] = "Happy belated birthday Amane!";
console.log(students);
// Delete
students.splice(0,2);
console.log(students);

// JS arrow function: () => {}
students.forEach((student) => {
  console.log(`${student} is great!`);
});

File: array.rb
# ARRAY CRUD
students = ['Steven', 'Tom', 'Erika']

# Create
students << "Kim" # only ruby
students.push("Amane")
p students
# Read
p students[3]
# Update
students[4] = "Happy belated birthday Amane!"
p students
# Delete
students.delete("Steven")
p students
students.delete_at(1)
p students

students.each do |student|
  puts "#{student} is great!"
end